### ISSUE TITLE
Detect flaky tests by executing failed tests mutliple times

### ISSUE THREAD

**Issue by sglahn** [rewe-digital-incubator/frost#11]
It would be nice to have the ability to detect flaky tests and/or have flaky tests not break the build. 
This can be achieved by executing failed tests multiple times, e.g. 3 times, and only break the build if the test has failed every single time. If the test is successful, even only once, the build should be green but the test should be reported as flaky.
